src/libpriv/rpmostree-container.cxx

```diff
@@ -59,11 +60,10 @@ rpmostree_container_rebuild (rpmostreecxx::Treefile &treefile, GCancellable *can
   if (!dnf_context_run (dnfctx, NULL, error))
     return FALSE;
 
-  glnx_autofd int rootfs_fd = -1;
-  if (!glnx_opendirat (AT_FDCWD, "/", TRUE, &rootfs_fd, error))
-    return FALSE;
-  if (!rpmostree_cleanup_leftover_rpmdb_files (rootfs_fd, cancellable, error))
```

Comment:
Before, we were doing this unconditionally, but now it's conditional on passing `-L`. Should we just call `rpmostreecxx::postprocess_cleanup_rpmdb` if `!autolayer`?

Reply:
Yes, good catch!